client/components/editor/ActivityContent/index.vue

```diff
@@ -76,7 +76,8 @@ export default {
       return transform(elements, (acc, it) => {
         const comments = this.getComments({ activityId, contentElementId: it.id });
         const lastSeen = max([seen.contentElement[it.uid], seen.activity[activityUid]]);
-        acc[it.uid] = { ...it, comments, lastSeen: lastSeen || 0 };
+        const hasUnresolvedComments = !comments.length;
```

Comment:
missing one `!` (since getComments is returning unresolved comments) ? 🤔 
@lukasaric @kronicker ?